daemon/daemon.go

```diff
@@ -266,10 +266,16 @@ func Start(sockFile string, c *cli.Context) error {
 	config := &daemonConfig{
 		Root: root,
 	}
-	exists, err := util.ObjectExists(config)
-	if err != nil {
-		return err
+
+	ignoreCfgFile := c.Bool("ignore-config-file")
```

Comment:
The problem I encountered is that when I changed the command line options for starting the daemon, the daemon would always load config file and ignore the command line. For example, if I started daemon without  --ignore-docker-delete for the first time, then when I stopped the daemon and started it again but with the option specified. The option would not take effect.